diff --git a/CMakeLists.txt b/CMakeLists.txt
index 718a42c9..fa30520d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -61,6 +61,7 @@ if(BUILD_SDDM_THEME)
     DIRECTORY sddm-theme/
     DESTINATION ${KDE_INSTALL_FULL_DATADIR}/sddm/themes/breath
     PATTERN "README.txt" EXCLUDE
+    PATTERN "components" EXCLUDE
     PATTERN "dummydata" EXCLUDE
     PATTERN "theme.conf.cmake" EXCLUDE)
 
diff --git a/lnf/breath-dark/contents/splash/Splash.qml b/lnf/breath-dark/contents/splash/Splash.qml
index 45b4b533..63c3a498 100644
--- a/lnf/breath-dark/contents/splash/Splash.qml
+++ b/lnf/breath-dark/contents/splash/Splash.qml
@@ -32,7 +32,7 @@ Image {
 
             anchors.centerIn: parent
             asynchronous: true
-            source: "images/manjarologo.svgz"
+            source: "images/fedora_logo_darkbackground.svg"
             sourceSize.height: size
         }
 
diff --git a/lnf/breath/contents/splash/Splash.qml b/lnf/breath/contents/splash/Splash.qml
index 45b4b533..63c3a498 100644
--- a/lnf/breath/contents/splash/Splash.qml
+++ b/lnf/breath/contents/splash/Splash.qml
@@ -32,7 +32,7 @@ Image {
 
             anchors.centerIn: parent
             asynchronous: true
-            source: "images/manjarologo.svgz"
+            source: "images/fedora_logo_darkbackground.svg"
             sourceSize.height: size
         }
 
diff --git a/sddm-theme/theme.conf.cmake b/sddm-theme/theme.conf.cmake
index fecbeec4..b915373d 100644
--- a/sddm-theme/theme.conf.cmake
+++ b/sddm-theme/theme.conf.cmake
@@ -1,7 +1,7 @@
 [General]
 showlogo=hidden
 showClock=true
-logo=${KDE_INSTALL_FULL_DATAROOTDIR}/sddm/themes/breath/manjarologo.svgz
+logo=${KDE_INSTALL_FULL_DATAROOTDIR}/sddm/themes/breath/fedora_logo_darkbackground.svg
 type=image
 color=#1abc9c
 fontSize=10
