--- rmux-ipc-0.3.1/Cargo.toml	2006-07-24T01:21:28+00:00
+++ rmux-ipc-0.3.1/Cargo.toml	2026-05-25T13:28:41.429400+00:00
@@ -83,13 +83,3 @@
     "net",
 ]
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.2"
-features = [
-    "Win32_Foundation",
-    "Win32_Security",
-    "Win32_Security_Authorization",
-    "Win32_Storage_FileSystem",
-    "Win32_System_Pipes",
-    "Win32_System_Threading",
-]
