diff --git a/deno/rust-libsui/rust-libsui.spec b/deno/rust-libsui/rust-libsui.spec
index a2a22a0..5528d10 100644
--- a/deno/rust-libsui/rust-libsui.spec
+++ b/deno/rust-libsui/rust-libsui.spec
@@ -16,6 +16,7 @@ Source:         %{crates_source}
 Patch:          libsui-fix-metadata-auto.diff
 
 BuildRequires:  cargo-rpm-macros >= 24
+BuildArch:      x86_64
 
 %global _description %{expand:
 A injection tool for executable formats (ELF, PE, Mach-O) that allows
@@ -39,7 +40,6 @@ License:        # FIXME
 
 %package        devel
 Summary:        %{summary}
-BuildArch:      noarch
 
 %description    devel %{_description}
 
@@ -53,7 +53,6 @@ use the "%{crate}" crate.
 
 %package     -n %{name}+default-devel
 Summary:        %{summary}
-BuildArch:      noarch
 
 %description -n %{name}+default-devel %{_description}
 
