--- rmux-os-0.3.0/Cargo.toml	2006-07-24T01:21:28+00:00
+++ rmux-os-0.3.0/Cargo.toml	2026-05-24T16:06:36.332778+00:00
@@ -51,17 +51,3 @@
     "termios",
 ]
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.2"
-features = [
-    "Win32_Foundation",
-    "Win32_Security",
-    "Win32_Security_Authorization",
-    "Win32_Storage_FileSystem",
-    "Win32_System_Console",
-    "Win32_System_Diagnostics_Debug",
-    "Win32_System_Memory",
-    "Win32_System_SystemInformation",
-    "Win32_System_Threading",
-    "Wdk_System_Threading",
-]
