--- boringtun-0.6.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ boringtun-0.6.0/Cargo.toml	2026-05-21T14:44:03.543522+00:00
@@ -77,7 +77,7 @@
 features = ["getrandom"]
 
 [dependencies.ring]
-version = "0.16"
+version = "0.17"
 
 [dependencies.socket2]
 version = "0.4.7"
@@ -100,15 +100,11 @@
 version = "0.9.0"
 
 [dependencies.x25519-dalek]
-version = "=2.0.0-rc.3"
+version = "2.0.1"
 features = [
     "reusable_secrets",
     "static_secrets",
 ]
-
-[dev-dependencies.criterion]
-version = "0.3.5"
-features = ["html_reports"]
 
 [dev-dependencies.etherparse]
 version = "0.12"
@@ -130,7 +126,7 @@
 mock-instant = ["mock_instant"]
 
 [target."cfg(unix)".dependencies.nix]
-version = "0.25"
+version = "0.26"
 features = [
     "time",
     "user",
