diff --git a/src/tui.rs b/src/tui.rs
index e2c22ff..b056dd3 100644
--- a/src/tui.rs
+++ b/src/tui.rs
@@ -3411,7 +3411,10 @@ impl<'a> NodeMap<'a> {
         let shift = 64 - width;
         let mask = (slots - 1) as u64;
 
-        let slots = arena.alloc_uninit_slice(slots).write_filled(None);
+        let slots = arena.alloc_uninit_slice(slots);
+        slots.fill(std::mem::MaybeUninit::new(None));
+        let slots = unsafe { slots.assume_init_mut() };
+
         let mut node = tree.root_first;
 
         loop {
