--- tendril-0.5.0/Cargo.toml	1970-01-01T00:00:01+00:00
+++ tendril-0.5.0/Cargo.toml	2026-03-20T15:57:46.860854+00:00
@@ -37,10 +37,6 @@
 name = "tendril"
 path = "src/lib.rs"
 
-[[example]]
-name = "fuzz"
-path = "examples/fuzz.rs"
-
 [[bench]]
 name = "futf"
 path = "benches/futf.rs"
@@ -60,9 +56,3 @@
 
 [dependencies.utf-8]
 version = "0.7"
-
-[dev-dependencies.criterion]
-version = "0.8"
-
-[dev-dependencies.rand]
-version = "0.9"
